ldapAdapterEXT-ext-impl/src/main/resources/ldapimporter.properties

```diff
@@ -0,0 +1,24 @@
+# properties for CustomPortalLDAPImporterImpl
+
+## if the system.config.path is set, then the system-wide configuration will override settings found in this file
+#system.config.path=
+
+## when true, importer will not add user to any organizations if no org mapping has matched
+## otherwise, the organization gets searched or created as it is, without mapping
+## defaults to false
+# default.to.none=true
```

Comment:
Where do you set the default organization, to which the user is mapped, if `defaults.to.none=false`?